project.clj

```diff
@@ -3,8 +3,8 @@
   :license      {:name "MIT"
                  :url "https://github.com/CoNarrative/precept/blob/master/LICENSE"}
   :dependencies [[org.clojure/clojure "1.9.0-alpha15"]
-                 [org.clojure/clojurescript "1.9.494"]
-                 [precept "0.3.0-alpha"]
+                 [org.clojure/clojurescript "1.9.854"]
```

Comment:
Looks like this is the latest and greatest? Probably want to update Precept to this version or replace this with the version Precept is using. The projects in the precept/examples folder will be affected by this too.